indico/modules/rb_new/views/base.py

```diff
@@ -25,7 +25,7 @@ class WPRoomBookingBase(WPNewBase):
     template_prefix = 'rb/'
     title = _('Room Booking')
     bundles = ('common.js', 'common.css', 'react.js', 'react.css', 'semantic-ui.js', 'semantic-ui.css',
-               'module_rb_new.js', 'module_rb_new.css')
+               'module_rb_new.js', 'module_rb_new.css', 'outdatedbrowser.js', 'outdatedbrowser.css')
```

Comment:
what about adding those two in `WPNewBase`? That way it's guaranteed to be included on all 'new' pages in case we add more in the future!

Reply:
moved it there since burotel inherits directly from `WPNewBase`